### ISSUE TITLE
Evaluate FTL (BNN) model with DFFD

### ISSUE THREAD

**Issue by pritshah9** [aravadikesh/DeepFakeDetector#10]
The media we want to classify with have manipulated or generated faces. We know the model works well with CIFAKE, we need to check it on DFFD as that is closer to the images we are targeting.

**Comment by pritshah9** [comment]
DFFD is compiled using multiple different subdatasets. I used CelebA (for real imgs) and PGGAN_v2 (for fakes).
The model works well with DFFD and achieves >0.95 accuracy on ~10k real images from celebA test imgs  and ~10k fake images from DFFD. I selected a 0.9 threshold for prediction through probability (after looking at the logits and corresponding prob). So we predict real if probability is >0.9 and fake otherwise. The accuracy on just the real imgs was 100% which makes this great for filtering out fakes as probability of incorrectly classifying real images is low.

Furthur threshold optimization, possibly with more data, can furthur improve performance.
